cli/run.go

```diff
@@ -71,7 +75,7 @@ func RunRun(r *cmd.RootCMD, c *cmd.CMD) {
 	// Load Triggers
 	tm, err := config.LoadAll()
 	if err != nil {
-		log.Fatalf("Failed to load triggers, reason: %s\n", err)
+		log.Panicf("Failed to load triggers, reason: %s\n", err)
```

Comment:
I'm not sure panicking is the right approach. Panic should be for programming errors, not users'.